playbooks/roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: 'Clone the hammer repositories'
+  git:
+    repo: https://github.com/{{ item }}.git
+    dest: ~/{{ item.split('/')[1] }}
+    update: no
+  with_items: "{{ hammer_devel_repositories }}"
+
+- name: 'Add local gems to Gemfile.local'
+  blockinfile:
+    dest: ~/hammer-cli-katello/Gemfile.local
+    create: yes
+    block: |
+      gem 'hammer_cli', :path => '../hammer-cli'
+      gem 'hammer_cli_foreman', :path => '../hammer-cli-foreman'
+      gem 'hammer_cli_foreman_tasks', :path => '../hammer-cli-foreman-tasks'
+
+- name: 'Remove hammer_cli_foreman in Gemfile'
```

Comment:
With https://github.com/Katello/hammer-cli-katello/pull/442, you can remove lines 18-28.